src/server/svr_func.c

```diff
@@ -3958,46 +3944,52 @@ set_entity_resc_sum_max(job *pjob, pbs_queue *pque, attribute *altered_resc,
-		ret_error = rc;
-		snprintf(log_buffer, LOG_BUF_SIZE-1, "set_entity_ct_sum_max %s on %s %s failed with %d",
-				(op == INCR) ? "INCR" : "DECR", pque?"queue":"server", pque?pque->qu_qs.qu_name:server_name, rc);
-		log_event(PBSEVENT_ERROR, PBS_EVENTCLASS_JOB, LOG_NOTICE, pjob->ji_qs.ji_jobid, log_buffer);
-	}
+	if ((op_flag & ETLIM_ACC_CT_MAX) == ETLIM_ACC_CT_MAX)
+		if ((rc = set_entity_ct_sum_max(pjob, pque, op)) != 0) {
+			ret_error = rc;
+			snprintf(log_buffer, LOG_BUF_SIZE-1, "set_entity_ct_sum_max %s on %s %s failed with %d",
+					(op == INCR) ? "INCR" : "DECR", pque?"queue":"server", pque?pque->qu_qs.qu_name:server_name, rc);
+			log_event(PBSEVENT_ERROR, PBS_EVENTCLASS_JOB, LOG_NOTICE, pjob->ji_qs.ji_jobid, log_buffer);
+		}
 
-	if ((rc = set_entity_ct_sum_queued(pjob, pque, op)) != 0) {
-		ret_error = rc;
-		snprintf(log_buffer, LOG_BUF_SIZE-1, "set_entity_ct_sum_queued %s on %s %s failed with %d",
-				(op == INCR) ? "INCR" : "DECR", pque?"queue":"server",pque?pque->qu_qs.qu_name:server_name, rc);
-		log_event(PBSEVENT_ERROR, PBS_EVENTCLASS_JOB, LOG_NOTICE, pjob->ji_qs.ji_jobid, log_buffer);
-	}
+	if((op_flag & ETLIM_ACC_CT_QUEUED) == ETLIM_ACC_CT_QUEUED)
+		if ((rc = set_entity_ct_sum_queued(pjob, pque, op)) != 0) {
+			ret_error = rc;
+			snprintf(log_buffer, LOG_BUF_SIZE-1, "set_entity_ct_sum_queued %s on %s %s failed with %d",
+					(op == INCR) ? "INCR" : "DECR", pque?"queue":"server",pque?pque->qu_qs.qu_name:server_name, rc);
+			log_event(PBSEVENT_ERROR, PBS_EVENTCLASS_JOB, LOG_NOTICE, pjob->ji_qs.ji_jobid, log_buffer);
+		}
 
-	if ((rc = set_entity_resc_sum_max(pjob, pque, altered_resc, op)) != 0) {
-		ret_error = rc;
-		snprintf(log_buffer, LOG_BUF_SIZE-1, "set_entity_resc_sum_max %s on %s %s failed with %d, (altered_resc %p)",
-				(op == INCR) ? "INCR" : "DECR", pque?"queue":"server",pque?pque->qu_qs.qu_name:server_name, rc, altered_resc);
-		log_event(PBSEVENT_ERROR, PBS_EVENTCLASS_JOB, LOG_NOTICE, pjob->ji_qs.ji_jobid, log_buffer);
-	}
+	if((op_flag & ETLIM_ACC_RES_MAX) == ETLIM_ACC_RES_MAX)
+		if ((rc = set_entity_resc_sum_max(pjob, pque, altered_resc, op)) != 0) {
+			ret_error = rc;
+			snprintf(log_buffer, LOG_BUF_SIZE-1, "set_entity_resc_sum_max %s on %s %s failed with %d, (altered_resc %p)",
+					(op == INCR) ? "INCR" : "DECR", pque?"queue":"server",pque?pque->qu_qs.qu_name:server_name, rc, altered_resc);
+			log_event(PBSEVENT_ERROR, PBS_EVENTCLASS_JOB, LOG_NOTICE, pjob->ji_qs.ji_jobid, log_buffer);
+		}
 
-	if ((rc = set_entity_resc_sum_queued(pjob, pque, altered_resc, op)) != 0) {
-		ret_error = rc;
-		snprintf(log_buffer, LOG_BUF_SIZE-1, "set_entity_resc_sum_queued %s on %s %s failed with %d, (altered_resc %p)",
-				(op == INCR) ? "INCR" : "DECR", pque?"queue":"server",pque?pque->qu_qs.qu_name:server_name, rc, altered_resc);
-		log_event(PBSEVENT_ERROR, PBS_EVENTCLASS_JOB, LOG_NOTICE, pjob->ji_qs.ji_jobid, log_buffer);
-	}
+	if((op_flag & ETLIM_ACC_RES_QUEUED) == ETLIM_ACC_RES_QUEUED)
```

Comment:
invoke `set_entity_resc_sum_queued()` only if `ETLIM_ACC_RES_QUEUED`  is set i.e  both `(ETLIM_ACC_RES | ETLIM_ACC_QUEUED)`